packages/rum-core/src/domain/rumEventsCollection/action/actionCollection.ts

```diff
@@ -44,6 +44,7 @@ function processAction(action: AutoAction | CustomAction) {
   const actionEvent = combine(
     {
       action: {
+        id: generateUUID(),
```

Comment:
What about generating/adding it only for custom actions?

Reply:
I'm probably missing a point. I don't really see why we only have to target custom actions?

Reply:
action id for auto action is generated earlier in the code and retrieved in `autoActionProperties`.
With the current code, for an auto action, an id is generated and overwritten immediately by the id from `autoActionProperties`.

IMO, we could make the intent clearer by having something like:
```ts
const actionTypeProperties = isAutoAction(action)
    ? {
        action: {
          error: {
            count: action.counts.errorCount,
          },
          id: action.id,
          loading_time: toServerDuration(action.duration),
          long_task: {
            count: action.counts.longTaskCount,
          },
          resource: {
            count: action.counts.resourceCount,
          },
        },
      }
    : {
        action: {
          id: generateUUID()
        }
      }
```